src/axom/sidre/core/DataStore.cpp

```diff
@@ -129,6 +129,40 @@ DataStore::~DataStore()
   }
 }
 
+/*
+ *************************************************************************
+ *
+ * Rewire Conduit Message Handlers to SLIC.
+ *  ... Not the best function name, but it is descriptive :-) 
+ *
+ *************************************************************************
+ */
+void DataStore::wireConduitMessageHandlersToSLIC()
```

Comment:
I'm ok with `wire`, but as an alternate suggestion to complement `reset...` below, how about just `setConduitMessageHandlersToSLIC()` ?

Reply:
Leaning towards `set`